packages/server/lib/modes/record.js

```diff
@@ -228,7 +227,7 @@ const updateInstance = (options = {}) => {
       error,
       video,
       hooks,
-      stdout,
+      stdout: null,
```

Comment:
does the schema prevent it from being `undefined`?

Reply:
Yep, has to be string or null

Reply:
let's leave a comment indicating why this change is necessary